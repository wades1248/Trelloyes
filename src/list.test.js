import React from 'react';
import ReactDOM from 'react-dom';
import List from './list';
import renderer from 'react-test-renderer';

describe('list component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<List header='list' cards={[{},{}]} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders UI as expected', () => {
        const tree = renderer
            .create(<List header='list' cards={[{},{}]} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});