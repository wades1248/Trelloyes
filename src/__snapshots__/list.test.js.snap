// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list component renders UI as expected 1`] = `
<section
  className="List"
>
  <header>
    <h2>
      list
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <div
      className="Card"
    >
      <h3 />
      <p />
    </div>
    <div
      className="Card"
    >
      <h3 />
      <p />
    </div>
  </div>
</section>
`;
